lib/streamlit/type_util.py

```diff
@@ -440,6 +444,11 @@ def convert_anything_to_df(df: Any) -> DataFrame:
     if is_type(df, "numpy.ndarray") and len(df.shape) == 0:
         return pd.DataFrame([])
 
+    if is_type(df, "snowflake.snowpark.dataframe.DataFrame") and not isinstance(
```

Comment:
nit: I think you can use the constant here: `_SNOWPARK_DF_TYPE_STR`

Reply:
It's updated 👍